scr/main/java/bittercode/util/DatabaseConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.bittercode.util;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+class DatabaseConfig {
+
+    static Properties prop = new Properties();
+    static {
+
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        InputStream input = classLoader.getResourceAsStream("application.properties");
+
+        try {
+            prop.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }

```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Improve exception handling</b></div>
<div id="fix">

Consider logging the exception or throwing a custom exception instead of just printing the stack trace. This would provide more context and allow for better error handling in the calling code.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -17,3 +17,3 @@
          } catch (IOException e) {
 -            e.printStackTrace();
 +            throw new RuntimeException("Failed to load application.properties", e);
          }
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434584858>#199df6</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly